go/parquet/pqarrow/schema.go

```diff
@@ -388,7 +388,7 @@ func fieldToNode(name string, field arrow.Field, props *parquet.WriterProperties
 		}
 		return schema.MapOf(field.Name, keyNode, valueNode, repFromNullable(field.Nullable), -1)
 	default:
-		return nil, xerrors.New("not implemented yet")
+		return nil, xerrors.Errorf("support for %v not implemented yet", field.Type.ID())
```

Comment:
let's use `fmt.Errorf("%w: support for %s", arrow.ErrNotImplemented, field.Type.ID())`

Reply:
Updated 👍